logstash-core/lib/logstash/runner.rb

```diff
@@ -93,6 +93,11 @@ class LogStash::Runner < Clamp::StrictCommand
     :attribute_name => "pipeline.workers",
     :default => LogStash::SETTINGS.get_default("pipeline.workers")
 
+  option ["--experimental-java-execution"], "COUNT",
```

Comment:
replace "COUNT" with `:flag` to correctly set the boolean flag.

```
ERROR: option '--experimental-java-execution'
```

I think we could drop the "experimental"

Reply:
@andrewvc wanted the `experimental` explicitly :) see https://github.com/elastic/logstash/pull/8357#issuecomment-332918877

But yea, let me fix that my bad. It works if you use `--experimental-java-execution=true` ... should've been a pointer for me to remove that i needed the `=true`:D